templates/mjs/www.ejs

```diff
@@ -5,7 +5,7 @@
  */
 
 import app from '../app.mjs';
-import http from 'node:http';
+import http from 'http';
```

Comment:
Grouping Node.js modules at the top of the file and Express import afterward is good practice, leaving a blank line between the two blocks.

```
import http from 'http';

import app from '../app.mjs';
```

Reply:
@aagamezl Thanks for the suggestion. Change has been made.